constantine/math/constants/banderwagon_sqrt.nim

```diff
@@ -154,4 +1496,4 @@ func precompute_tonelli_shanks_addchain*(
   # 261 + 3 = 264 operations
   r *= x11111111
   r.square()
-  r *= a
+  r *= a
```

Comment:
Files should end with empty-lines, normally vscode and vim/nvim do that automatically